openedx/core/djangoapps/discussion_common/comment_client/thread.py

```diff
@@ -112,19 +113,23 @@ def search(cls, query_params):
         )
 
     @classmethod
-    def url_for_threads(cls, params={}):
```

Comment:
Ugh, hadn't realized we still had mutable default args lingering in the codebase. :( Good conversion to `None`.